js_modules/dagit/packages/core/src/instance/BackfillTable.tsx

```diff
@@ -160,17 +144,13 @@ export const BackfillTable = ({
         backfill={partitionsRequestedBackfill}
         onClose={() => setPartitionsRequestedBackfill(undefined)}
       />
-      <BackfillTerminationDialog
-        backfill={terminationBackfill}
-        onClose={() => setTerminationBackfill(undefined)}
-        onComplete={() => refetch()}
-      />
-      <TerminationDialog
-        isOpen={!!unfinishedRuns?.length}
-        onClose={() => setCancelRunBackfill(undefined)}
-        onComplete={() => refetch()}
-        selectedRuns={unfinishedMap}
-      />
+      {terminationBackfill ? (
+        <BackfillTerminationDialog
+          backfill={terminationBackfill}
+          onClose={() => setTerminationBackfill(undefined)}
+          onComplete={() => refetch()}
+        />
+      ) : null}
```

Comment:
Can this always be allowed to render? As written, it will pop out of existence instead of animating closed when `onClose` occurs.

Reply:
yeah, it can... I can change that up